weed/shell/command_volume_check_disk.go

```diff
@@ -49,6 +49,8 @@ func (c *commandVolumeCheckDisk) Do(args []string, commandEnv *CommandEnv, write
 	slowMode := fsckCommand.Bool("slow", false, "slow mode checks all replicas even file counts are the same")
 	verbose := fsckCommand.Bool("v", false, "verbose mode")
 	applyChanges := fsckCommand.Bool("force", false, "apply the fix")
+	missingKeysThreshold := fsckCommand.Float64("missingKeysThreshold", 0.3, "repair when missing keys is not more than this limit")
```

Comment:
The parameter name is confusing whether it is a maximum limit or minimum limit without looking at the help message.

And it is confusing whether it is a percentage or not.

When this parameter is needed?

Reply:
> When this parameter is needed?
Since this command has been added to cronjob 
And it starts automatically, then I want to have protection against a possible software error in the case of an abnormal situation where the number of lost keys cannot exceed a reasonable limit.
If the limit is reached, it means that something went wrong and the job will end with an error and I can draw attention to this and come in manual mode to fix it
https://github.com/chrislusf/seaweedfs/blob/master/k8s/seaweedfs/templates/cronjob.yaml#L44

Reply:
need a better name than `missingKeysThreshold`

Reply:
> need a better name than `missingKeysThreshold`

@chrislusf 
`repairThreshold` norm ?

Reply:
`skipRepairThreshold`?

Reply:
> `skipRepairThreshold`?

In the sense then `stopRepairThreshold` 
Although from my point of view a pun
since Threshold already implies stopping the action

Reply:
The meanings of `Threshold` in `garbageThreshold` and `repairThreshold` are opposite.

Reply:
> The meanings of `Threshold` in `garbageThreshold` and `repairThreshold` are opposite.

nonRepairThreshold ?

Reply:
good!